codex/manifest/manifest.nim

```diff
@@ -30,17 +30,17 @@ export types
 
 type
   Manifest* = ref object of RootObj
-    treeCid {.serialize.}: Cid            # Root of the merkle tree
-    datasetSize {.serialize.}: NBytes     # Total size of all blocks
-    blockSize {.serialize.}: NBytes       # Size of each contained block (might not be needed if blocks are len-prefixed)
-    version: CidVersion                   # Cid version
-    hcodec: MultiCodec                    # Multihash codec
-    codec: MultiCodec                     # Data set codec
-    case protected {.serialize.}: bool    # Protected datasets have erasure coded info
+    treeCid: Cid           # Root of the merkle tree
```

Comment:
Was this intentional to remove `{.serialize.}`?

Reply:
ah, no - I guess this was mangled during the merge.